packages/prisma/migrations/20230303162003_add_booking_seat_reference/migration.sql

```diff
@@ -0,0 +1,34 @@
+-- DropForeignKey
+ALTER TABLE "Attendee" DROP CONSTRAINT IF EXISTS "Attendee_bookingId_fkey";
```

Comment:
This FK constraint was created by this branch, so this broke this migration whenever it was applied somewhere where this branch hadn't been applied to yet.